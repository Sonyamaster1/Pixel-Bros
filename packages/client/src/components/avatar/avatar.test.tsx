import * as renderer from 'react-test-renderer'
import { Avatar } from '.'

describe('тест для аватара', () => {
  let component: renderer.ReactTestRenderer

  beforeEach(() => {
    component = renderer.create(
      <Avatar
        src="/53ab8463-3462-48ae-a03a-56f6d1f46735/0a496edf-57c0-4e82-82e2-dd49572f2949_котенок с молоком.jpe"
        handleOnChangeField={() => console.log('test')}
      />
    )
  })
  it('create snapshot', () => {
    expect(component.toJSON()).toMatchSnapshot()
  })

  it('should render correct svg icon', () => {
    expect(
      component.root.findByProps({ 'data-testid': 'avatarImg' })
    ).toBeDefined()
  })
})
