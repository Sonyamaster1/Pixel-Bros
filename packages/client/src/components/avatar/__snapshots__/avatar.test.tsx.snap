// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`тест для аватара create snapshot 1`] = `
<div>
  <img
    alt="avatar"
    data-testid="avatarImg"
    src="https://ya-praktikum.tech/api/v2/resources/53ab8463-3462-48ae-a03a-56f6d1f46735/0a496edf-57c0-4e82-82e2-dd49572f2949_котенок с молоком.jpe"
  />
  <input
    accept="image/*"
    id="avatar"
    name="avatar"
    onChange={[Function]}
    type="file"
  />
</div>
`;
